src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1190,10 +1189,15 @@ private JSlider getJSliderHeight() {
 		if (jSliderHeight == null) {
 			jSliderHeight = new JSlider(ExaminationParameters.HEIGHT_MIN, ExaminationParameters.HEIGHT_MAX, ExaminationParameters.HEIGHT_INIT);
 			jSliderHeight.addChangeListener(e -> {
-				int value = jSliderHeight.getValue();
-				jTextFieldHeight.setText(String.valueOf(value));
-				patex.setPex_height(value);
-				updateBMI();
+				JSlider source = (JSlider)e.getSource();
+		        if (!source.getValueIsAdjusting()) {
+		        	System.out.println("change!");
```

Comment:
Not sure you want to leave this in production.

Reply:
Sure not! thanks!